src/libraries/Microsoft.PowerFx.Connectors/Execution/IConnectorInvoker.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Connectors.Execution
+{
+    public interface IConnectorInvoker
+    {
+        // runtime context
+        BaseRuntimeConnectorContext Context { get; }
+
+        // request: classic call
+        Task<FormulaValue> InvokeAsync(FormulaValue[] args, CancellationToken cancellationToken);
+
+        // resquest: support for paging
+        Task<FormulaValue> InvokeAsync(string nextlink, CancellationToken cancellationToken);        
+        
+        // processing/inner call
+        Task<FormulaValue> SendAsync(InvokerParameters invokerElements, CancellationToken cancellationToken);
+    }
+
+    public class InvokerParameters
```

Comment:
>InvokerParameters [](http://example.com/codeflow?start=17&length=17)

This is effectively our HttpRequestMessage abstraction. 

It's actually useful to have this.

Reply:
Yes.